app/controllers/users.js

```diff
@@ -0,0 +1,17 @@
+const users = require('../models/user').users;
+const validations = require('./validations');
+const bcrypt = require('bcryptjs');
+
+module.exports = {
+  create(req, res) {
+    return users
+      .create({
+        firstName: req.users.firstName,
+        lastName: req.lastName.lastName,
+        email: req.users.email,
+        passowrd: bcrypt.hashSync(req.users.passowrd, 10)
```

Comment:
- You have a Typo in 'password'
- What happen if hashSync fail for any reason ? 
- Use a constant for 10 (in a constants file maybe)